pkg/0e89154b-a448-4fea-a115-169fb56c0cb9/package.lp

```diff
@@ -0,0 +1,185 @@
+   (vertex (position -0.65 0.15) (angle 0.0))
+   (vertex (position -0.65 -0.15) (angle 0.0))
+   (vertex (position -1.05 -0.15) (angle 0.0))
+   (vertex (position -1.05 0.15) (angle 0.0))
+  )
+  (polygon e7b10b4d-e78c-4017-8eca-b8b6093763c8 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area true)
+   (vertex (position -1.05 -0.5) (angle 0.0))
+   (vertex (position -0.65 -0.5) (angle 0.0))
+   (vertex (position -0.65 -0.8) (angle 0.0))
+   (vertex (position -1.05 -0.8) (angle 0.0))
+   (vertex (position -1.05 -0.5) (angle 0.0))
+  )
+  (polygon 11f9e3f5-34c4-4420-9930-686e2fba875a (layer top_documentation)
+   (width 0.0) (fill true) (grab_area true)
+   (vertex (position -1.05 0.8) (angle 0.0))
+   (vertex (position -0.65 0.8) (angle 0.0))
+   (vertex (position -0.65 0.5) (angle 0.0))
+   (vertex (position -1.05 0.5) (angle 0.0))
+   (vertex (position -1.05 0.8) (angle 0.0))
+  )
+  (polygon ab78ccbf-0862-443d-8818-7b3c32f4f7ee (layer top_documentation)
+   (width 0.0) (fill true) (grab_area true)
+   (vertex (position 1.05 -0.5) (angle 0.0))
+   (vertex (position 0.65 -0.5) (angle 0.0))
+   (vertex (position 0.65 -0.8) (angle 0.0))
+   (vertex (position 1.05 -0.8) (angle 0.0))
+   (vertex (position 1.05 -0.5) (angle 0.0))
+  )
+  (polygon ee02f3d4-d28e-40ec-a120-d04bd345a210 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area true)
+   (vertex (position 1.05 0.8) (angle 0.0))
+   (vertex (position 0.65 0.8) (angle 0.0))
+   (vertex (position 0.65 0.5) (angle 0.0))
+   (vertex (position 1.05 0.5) (angle 0.0))
+   (vertex (position 1.05 0.8) (angle 0.0))
+  )
+  (stroke_text 2bc00585-517c-4ab7-b4e0-ab277a4625b7 (layer top_names)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center bottom) (position 0.0 1.905) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{NAME}}")
+  )
+  (stroke_text 2ae8b25f-66a8-4875-8b97-f274ec01c554 (layer top_values)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center top) (position 0.0 -1.905) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{VALUE}}")
+  )
+ )
+ (footprint b6b8935d-b11c-4c95-b843-d6b415149a20
+  (name "hand soldering")
+  (description "")
+  (pad 34f43b73-db56-4649-8904-0d258eeb7bfe (side top) (shape rect)
+   (position -1.2 0.75) (rotation 0.0) (size 1.2 0.6) (drill 0.0)
+  )
+  (pad a2b521d2-b58b-45b4-944a-65bd6e6e158a (side top) (shape rect)
+   (position -1.2 0.0) (rotation 0.0) (size 1.2 0.4) (drill 0.0)
+  )
+  (pad fa0eed07-e0ca-4ed6-bb33-4c54a46bb925 (side top) (shape rect)
+   (position -1.2 -0.75) (rotation 0.0) (size 1.2 0.6) (drill 0.0)
+  )
+  (pad dc18d314-08ec-45f8-bd81-8b83857d1402 (side top) (shape rect)
+   (position 1.2 -0.75) (rotation 0.0) (size 1.2 0.6) (drill 0.0)
+  )
+  (pad a837e245-00a8-45f7-befe-8aaafc1001f8 (side top) (shape rect)
+   (position 1.2 0.75) (rotation 0.0) (size 1.2 0.6) (drill 0.0)
```

Comment:
Aren't these pads for hand soldering too large? I would say it should be enough to make them just a little bit larger than the reflow soldering pads.

Reply:
Note that they're 1.2x0.6mm with the large variant (with about 0.8mm sticking out under the lead), that's still tiny! Do you still think they should be made smaller?

Reply:
I'm pretty sure that 1.0x0.6mm should be enough for hand soldering (that's already 67% larger than the reflow solder pads).

Reply:
Since we explicitly have footprint versions with larger pads for hand soldering, why not make them actually easy to solder by hand? :slightly_smiling_face:  People that want space-efficient footprints will use the reflow version anyways.

But OK, I resized the pads to 1mm length.

Reply:
1.0x0.6mm might sound very small, but I guess this should be *easily* solderable by hand. And in my experience, space is nearly always a problem, so compact footprints are always a good thing :smiley: If there is still need for larger pads, we can also add a "hand soldering XL" footprint :grin: